### ISSUE TITLE
EF lazy-loading for related data

### ISSUE THREAD

**Issue by LouisKamp** [graphql-aspnet/graphql-aspnet#26]
Just came across this project and it looks awesome!
Im trying query lazy-loaded related data on a ef-context, but im receiving `null`. (You can have a look at my code [here](https://github.com/LouisKamp/graphql-aspnet-test)).

Is there a way to fix this apart from creating a type extension?

My model:
```c#

public class Blog
    {
        public int BlogId { get; set; }
        public string Url { get; set; }

        public virtual IList<Post> Posts { get; set; }
    }

    public class Post
    {
        public int PostId { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }

        public int BlogId { get; set; }
        public virtual Blog Blog { get; set; }
    }

```

When querying Posts on a Blog, the posts field will be `null`: 
```graphql
query {
  blogs {
    blogId
    url
    posts {
      content
    }
  }
}
```
Above returns:
```json
{
  "data": {
    "blogs": [
      {
        "blogId": 1,
        "url": "test",
        "posts": null
      }
    ]
  }
}
```

**Comment by kevin-carroll** [comment]
Hey Louis!

Yes it should walk `blogs -> posts collection -> post properties` and return the expected data without a type extension.  At first glance sounds like a bug.  Give me a few days to investigate I'll get back to you!  

Thanks a ton for the sample project, it really speeds up the process :)

**Comment by kevin-carroll** [comment]
Bug fixed with PR #27 

Deployed with `v0.6.1-beta`
https://www.nuget.org/packages/GraphQL.AspNet/
